spiffworkflow-backend/src/spiffworkflow_backend/routes/script_assist_controller.py

```diff
@@ -0,0 +1,54 @@
+from os import environ
+from flask import make_response
+from flask import request
+from flask.wrappers import Response
+from openai import OpenAI
+
+# TODO: We could just test for the existence of the API key, if it's there, it's enabled.
+# Leaving them separate now for clarity.
+# Note there is an async version in the openai lib if that's preferable.
+def enabled() -> Response:
+    response = environ.get("SPIFFWORKFLOW_SCRIPT_ASSIST_ENABLED");
```

Comment:
Please use current_app.config instead of environ to make sure we use the loaded configurations instead of env vars.

Reply:
Done.

Reply:
This change breaks it. The environment variables aren't found.